File: sdks/python/coop.py
"""Compatibility imports for the pre-Rookhold ``coop`` module name.

New code should import :mod:`rookhold`.  The aliases remain available through
the v0.6 migration so existing applications can upgrade the distribution
without changing their imports atomically.
"""

import rookhold as _rookhold
from rookhold import *  # noqa: F403

__version__ = _rookhold.__version__

Coop = _rookhold.Rookhold
CoopError = _rookhold.RookholdError
CoopEvent = _rookhold.RookholdEvent
HashedCoopEvent = _rookhold.HashedRookholdEvent

File: sdks/typescript/coop.ts
/** Compatibility exports for the pre-Rookhold TypeScript API names. */

export * from "./rookhold.js";
export { Rookhold as Coop, RookholdError as CoopError } from "./rookhold.js";
export type {
  RookholdErrorInit as CoopErrorInit,
  RookholdEvent as CoopEvent,
  HashedRookholdEvent as HashedCoopEvent,
} from "./rookhold.js";
